sass/layouts/_about.scss

```diff
@@ -106,6 +106,18 @@
   height: 100%;
 }
 
+
+// Fix for firefox layer order issue
+.parallax-scene {
+  .layer {
+    pointer-events: none;
+  }
+  #landing-register {
```

Comment:
It's the only way to select only register button.
Since the button doesn't has its own class, but has just a Id.